.chloggen/munir_update-collector-builder-readme.yaml

```diff
@@ -0,0 +1,7 @@
+change_type: enhancement
```

Comment:
I don't really think this needs a changelog since this is only doc change. You can remove this.